ReleaseProcessAutomation/Jira/JiraFunctionality.cs

```diff
@@ -55,6 +61,39 @@ public void CreateAndReleaseJiraVersion (SemanticVersion currentVersion, Semanti
     ReleaseVersion(currentVersionID, nextVersionID, squashUnreleased);
   }
 
+  public bool MoveClosedIssuesToNewVersion (SemanticVersion currentVersion, SemanticVersion nextVersion)
+  {
+    var currentJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(currentVersion.ToString());
+    var nextJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(nextVersion.ToString());
+    if(currentJiraVersion == null || nextJiraVersion == null)
+    {
+      _console.WriteLine($"Could not find current jira version '{currentVersion}' or next jira version '{nextVersion}'");
+      return false;
+    }
+    
+    var closedIssues = _jiraIssueService.FindAllClosedIssues(currentJiraVersion.id);
+    _jiraIssueService.MoveIssuesToVersion(closedIssues, currentJiraVersion.id, nextJiraVersion.id);
+    return true;
+  }
+
+  public IEnumerable<JiraToBeMovedIssue> GetClosedVersionsSummaryWithExactFixVersion (SemanticVersion version)
+  {
+    var currentVersions = _jiraVersionCreator.FindAllVersionsWithVersionNumber(version.ToString()).ToList() 
```

Comment:
We only use `currentVersions` to check if an element is in that list. So instead we could use a `HashSet<T>` which would simplify the removal of the `fullVersion` and also the checking logic. Since `JiraProjectVersion` does not implement equality we would need to use the id instead. Also there is a `.ToHashSet` method ;)